test/salt/states/cron.sls

```diff
@@ -0,0 +1,4 @@
+
+cron_package:
```

Comment:
great work @daks! but isn't it the logrotate formula? ;)

Reply:
This state is added for Suse platforms only at kitchen run, because Suse image does not seems to have cron installed. As logrotate directly depends on cron running, test fails :)

Reply:
@javierbertoli Would it be worth adding `cron` to the Suse pre-salted images?